src/Router.js

```diff
@@ -8,14 +12,52 @@ const LandingPage = lazy(() => import('./Components/LandingPage/LandingPage'));
 const CreateImageWizard = lazy(() =>
   import('./Components/CreateImageWizard/CreateImageWizard')
 );
+const INVENTORY_TOTAL_FETCH_URL = '/api/inventory/v1/hosts';
 
 export const Router = () => {
+  const [hasSystems, setHasSystems] = useState(true);
+
+  useEffect(() => {
+    try {
+      axios
+        .get(`${INVENTORY_TOTAL_FETCH_URL}?page=1&per_page=1`)
+        .then(({ data }) => {
+          setHasSystems(data.total > 0);
+        });
+    } catch (e) {
+      /*eslint-disable no-console*/
+      console.log(e);
+      /*eslint-enable no-console*/
+    }
+  }, [hasSystems]);
+
   return (
-    <Routes>
-      <Route path="*" element={<LandingPage />}>
-        <Route path="imagewizard/:composeId?" element={<CreateImageWizard />} />
-        <Route path="share/:composeId" element={<ShareImageModal />} />
-      </Route>
-    </Routes>
+    <Suspense
+      fallback={
+        <Bullseye>
+          <Spinner size="xl" />
+        </Bullseye>
+      }
+    >
+      {!hasSystems ? (
```

Comment:
I think we should show the zero state based on whether or not the user has images, not systems.